src/main/java/edu/ucdavis/dss/ipa/api/components/budget/BudgetViewController.java

```diff
@@ -144,6 +146,30 @@ public LineItem createLineItem(@PathVariable long budgetScenarioId,
         return lineItem;
     }
 
+    @RequestMapping(value = "/api/budgetView/budgets/{budgetId}/instructorTypes", method = RequestMethod.POST, produces="application/json")
+    @ResponseBody
+    public InstructorType createLineItem(@PathVariable long budgetId,
```

Comment:
Should this be named 'createLineItem' if it returns 'InstructorType' and is at '/instructorTypes'? Perhaps createInstructorLineItem? Presumably there are other methods that also create line items?